packages/devtools_app/lib/src/framework/release_notes/release_notes.dart

```diff
@@ -188,13 +191,21 @@ class ReleaseNotesController extends SidePanelController {
     return;
   }
 
+  /// Convert all site-base-url relative links in [markdownContent]
+  /// to absolute links from the specified [baseUrl].
```

Comment:
nit: could you add an example transformation to this dart doc? That would help with maintaining this code moving forward if things were to change.